docker/Dockerfile.local

```diff
@@ -14,6 +14,6 @@ RUN mkdir build
 
 WORKDIR build
 RUN cmake -DCMAKE_INSTALL_PREFIX=/usr -DEXTERNAL_BOOST=$EXTERNAL -DEXTERNAL_MPI4PY=$EXTERNAL -DWITH_XTC=ON ..
-RUN make -j4 all
-RUN make install
-WORKDIR ../examples
+#RUN make -j4 all
+#RUN make install
+#WORKDIR ../examples
```

Comment:
Why?

Reply:
maybe to save time and see that building is working?